src/test/resources/checkstyle.xml

```diff
@@ -235,11 +235,6 @@ Change log:
         </module>
         <module name="NonEmptyAtclauseDescription"/>
         <module name="JavadocTagContinuationIndentation"/>
-        <module name="SummaryJavadoc">
-            <property name="forbiddenSummaryFragments"
-              value="^@return the *|^This method returns |^A [{]@code [a-zA-Z0-9]+[}]( is a )"/>
-        </module>
-        <module name="JavadocParagraph"/>
```

Comment:
reiterating comment from previous PR: 

I don't agree with requiring prefixing paragraphs in javadocs with <p> as adding value, instead, I think tedious requirements like that actually dissuade terse javadoc additions, and instead, leads to writing functions with empty javadocs, to "document in a later PR", which also means "never"

Reply:
without `<p>` to separate paragraphs, text will appear as one block (except when directly looking at the code)—e.g., generated HTML, IDE mouseover, etc.

the value of offering uniform, readable, and standardized documentation in a public-facing (open-source) project is hard to overestimate.

"document in a later PR" is actually an excellent task for someone learning about the project—whether someone at Salesforce or someone outside the company. (the ones currently in the code were mostly there for Javadocs that were missing when we started using Checkstyle.)

these requirements definitely feel tedious at first (several ZOS folks (including me!) have noted the same)—I hope that fades for you quickly.